### DISCUSSION TITLE
How to use with webpack and IE11

### DISCUSSION THREAD

**Discussion by skttl** [alpinejs/alpine#732] [4. General]
Hi

Anyone had any succes using this with a bundled js file (using webpack)?

I have an index.js with the following
```
import "alpinejs";
```

Which works perfectly in modern browsers. But breaks in IE11. with `Uncaught (in promise) undefined`

If I remove alpine from my bundle, and use the module/nomodule pattern with cdn urls, it works as expected (except for #714).

Is it not possible to have it in my own bundle and still support IE11?

**Comment by ryangjchandler** [comment]
@skttl - Have you tried importing like `import "alpinejs/dist/alpine-ie11.js"` - this will already have all of the polyfills etc available.

**Comment by skttl** [comment]
Just did - no luck.

But then again, do I really want to have all those polyfills in my bundle anyway? I think not - so I'll go back to the module/nomodule pattern :)

Thanks anyway 👍

**Reply by ryangjchandler** [reply]
@skttl - you would _need_ them if you wanted to support IE11 anyway?

**Reply by skttl** [reply]
Right, but modern browsers wouldnt download and run the nomodule script. They would, if it was included in my bundle.
